examples/ethernet_network.go

```diff
@@ -44,7 +49,7 @@ func main() {
 		}
 	}
 
-	ethernet_nw_id := "cc65605c-9a90-4293-8e53-4556d5cc7892"
+	ethernet_nw_id := "47f102d0-83d9-42fa-aaef-187906a4185b"
```

Comment:
try to avoid hardcoding uri/id and get them by name

Reply:
done